### ISSUE TITLE
convert "no readable configuration profiles found..." into welcome screen

### ISSUE THREAD

**Issue by chipitsine** [OpenVPN/openvpn-gui#115]
![ws](https://cloud.githubusercontent.com/assets/2217296/21742691/4fc2027a-d51d-11e6-80b8-5d763a0d7892.png)

I suggest the following approach

- one time is enough, no need to show that error more than once
- change text and add "Import" button to make this window looks like friendly welcome screen (e.g. https://docs.fedoraproject.org/en-US/Fedora//html/Installation_Guide/sect-installation-graphical-mode.html )

**Comment by selvanair** [comment]
Good idea. Any volunteers to add a friendly welcome screen that allows the user to set language, import configs etc.?

**Comment by glaubitz** [comment]
I just installed the OpenVPN client on Windows 11 and I'm seeing this dialog box as well with no obvious way how to actually create a configuration. After clicking "OK", there is no other GUI showing up despite the client continuing to run in the background.

Not sure what's supposed to happen here.

**Comment by gackt2** [comment]
Same here. Installed the community version on Windows 10, got the pop up and now OpenVPN doesn't open. =(
